csrc/preseg_passes/allocation_order_inference.cpp

```diff
@@ -108,6 +129,10 @@ void mapAllocationDomain(
     mapped_ids.pushBack(id);
   }
 
+  if (mapped_ids.empty()) {
+    return;
+  }
```

Comment:
Why is this necessary? The following code seems to be able to preserve the order of no IterDomains are mapped.

Reply:
You are right, this isn't necessary. I thought it's easier to read. But I can replace this with a comment.